### PR TITLE
Implement lovely designs from Ross

### PR DESCRIPTION
**Pull request by mermop**
[Designs](https://preview.uxpin.com/10c63ed6d0853182d7b242ba1bfa62600e8480df#/pages/116344697/simulate/no-panels?mode=f)

This involves:
- Remove Material - Ross's designs are much closer to vanilla Bootstrap
- Root to login instead of home
- Change buttons and cards
- Big visual changes to birth records and shares
- Include audit information in birth record show screen

Doesn't include:
- Changes to share create workflow - needs more work
- Changes to organisation member dashboard - there are other changes in #124, so they can be brought in there
- Favicon

### CODE REVIEW

File: app/views/user/birth_records/_modal_form.html.erb [hunk 1]
```diff
@@ -15,9 +15,7 @@
...
 
-<div class="text-center">
-  <a href="#" class="btn btn-primary btn-rounded mb-4" data-toggle="modal" data-target="#modalBirthRecordForm">
-    Search for Birth Record
-  </a>
-</div>
-
+<h3>Add a document</h3>
+<a href="#" data-toggle="modal" data-target="#modalBirthRecordForm" class="add-doc">
+  <%= image_tag("plus.svg") %>
```
[the remark this answers is not in the document]
Comment (mermop) [reply app/views/user/birth_records/_modal_form.html.erb]: Thanks mate

File: app/models/audit.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Audit < Audited::Audit
+  scope :viewing_birth_record, ->() { where(comment: AuditedOperationsService::VIEW_SHARED_BIRTH_RECORD) }
```
Comment (mermop) [inline app/models/audit.rb]: This is the biggest backend change - I'm a little uncomfortable about reaching into the service to get this constant, but I think it's probably the best at the moment

**Comment by Br3nda** [comment]
looks amazing on big screens

looks weird on small screens

image

**Comment by Br3nda** [comment]
is there a favicon too?

**Comment by mermop** [comment]
I'll need to grab files off Ross for the favicon - might be a job for tomorrow

**Comment by Br3nda** [comment]
only the edge of the logout is visible
image

**Comment by Br3nda** [comment]
i ran `pa11y` over this, and the colour contrast passed as accessible.

File: app/views/shared/_header.html.erb [hunk 3]
```diff
@@ -56,10 +53,10 @@
       <% if signed_in? %>
         <li class="nav-item mr-auto">
           <% if current_admin_user.present? %>
-            <%= render partial: 'shared/logout', locals: { _logout_path: destroy_admin_user_session_path, _account_badge_letter: 'A' } %>
+            <%= render partial: 'shared/logout', locals: { _logout_path: destroy_admin_user_session_path } %>
           <% end %>
           <% if current_user.present? %>
-            <%= render partial: 'shared/logout', locals: { _logout_path: destroy_user_session_path, _account_badge_letter: 'U' } %>
+            <%= render partial: 'shared/logout', locals: { _logout_path: destroy_user_session_path } %>
```
Comment (Br3nda) [inline app/views/shared/_header.html.erb]: why is it an `_` path?

Comment (Br3nda) [reply app/views/shared/_header.html.erb]: it's not new tho, so nvm

File: app/views/user/birth_records/_card.html.erb [hunk 4]
```diff
@@ -40,7 +40,9 @@
     <% end %>
   </div>
   <div class="card-footer">
-    <%= link_to 'share', new_user_share_path(share: { birth_record_id: record.to_param }), class: 'btn' %>
-    <%= link_to 'remove', remove_user_birth_record_path(record), class: 'btn btn-danger', data: { confirm: 'Are you sure? This will remove the record from your documents'}, method: :post %>
+    <div class="card-footer__inner">
```
Comment (Br3nda) [inline app/views/user/birth_records/_card.html.erb]: nice BEM

File: app/views/shared/_header.html.erb (hunk 3 above)
Comment (mermop) [reply app/views/shared/_header.html.erb]: Prefixing with `_` is a convention sometimes used to denote local variables in a partial

**Pull request merged**